rules/windows/process_creation/proc_creation_win_susp_hidden_dir_index_allocation.yml

```diff
@@ -0,0 +1,30 @@
+title: Potential Hidden Directory Creation Via NTFS INDEX_ALLOCATION Stream - CLI
+id: 0900463c-b33b-49a8-be1d-552a3b553dae
+related:
+    - id: a8f866e1-bdd4-425e-a27a-37619238d9c7
+      type: similar
+status: experimental
+description: |
+    Detects command line containing reference to the "::$index_allocation" stream. Which can be used as a technique to prevent access to folder and files from tooling such as "explorer.exe" and "cmd.exe"
```

Comment:
```suggestion
    Detects command line containing reference to the "::$index_allocation" stream, which can be used as a technique to prevent access to folders or files from tooling such as "explorer.exe" or "cmd.exe"
```